classes/ObjectModel.php

```diff
@@ -755,34 +759,35 @@ public function update($null_values = false)
             $this->id_shop_default = (in_array(Configuration::get('PS_SHOP_DEFAULT'), $id_shop_list) == true) ? Configuration::get('PS_SHOP_DEFAULT') : min($id_shop_list);
         }
         // Database update
-        if (!$result = Db::getInstance()->update($this->def['table'], $this->getFields(), '`' . pSQL($this->def['primary']) . '` = ' . (int) $this->id, 0, $null_values)) {
+        $updatedFields = $this->getFields();
```

Comment:
I think this should be either "fieldsToUpdate" (fields being the ones in the database) or "updatedProperties" (properties being the ones in ObjectModel)

Reply:
Ok I'll go with `fieldsToUpdate` here, so that it makes more sense with the `getFields` Mixing fields and properties would make the code more complex to read/understand